index.js

```diff
@@ -1,8 +1,17 @@
 'use strict';
 const path = require('path');
 const childProcess = require('child_process');
+const util = require('util');
 const isWsl = require('is-wsl');
 
+// Converts a path from WSL format to Windows format
+// e.g. /mnt/c/Program Files/Example/MyApp.exe
+//   => C:\Program Files\Example\MyApp.exe
+const wslToWindowsPath = async path => {
+	const {stdout} = await util.promisify(childProcess.execFile)('wslpath', ['-w', path]);
```

Comment:
Can you use a long flag version of `-w`? For readability.

Reply:
Better to promisify at the top-level instead of every time it's called.

```js
const pExecFile = util.promisify(childProcess.execFile);
```

Reply:
Unfortunately `wslpath` doesn't have any long-form flags.